library-best-practices/JLBP-12.md

```diff
@@ -0,0 +1,8 @@
+[JLBP-12] Make level of support and level of API stability clear
+----------------------------------------------------------------
+
+- Surface stability should be clearly described.
+- Feedback mechanisms should be outlined (for example, file an issue).
+- Clearly describe when a library is abandoned.
```

Comment:
This one is probably worth a JLBP of its own. &lt;cough>appengine-gcs-client&lt;/cough>